dind/docker-lib.sh

```diff
@@ -38,6 +38,11 @@ sanitize_cgroups() {
             ln -s "$mountpoint" "/sys/fs/cgroup/$sys"
         fi
     done
+
+    if [ ! -e /sys/fs/cgroup/systemd ] && [ $(cat /proc/self/cgroup | grep '^1:name=openrc:' | wc -l) -eq 0 ]; then
+      mkdir /sys/fs/cgroup/systemd
+      mount -t cgroup -o none,name=systemd none /sys/fs/cgroup/systemd
+    fi
```

Comment:
I needed this from latest version of script, in order to mount systemd path.  This allows dockerd to startup correctly with version 18.x